src/solCompile.ts

```diff
@@ -97,13 +102,19 @@ function cliCompile(
     allowedPaths = `--allow-paths ${currentDirectory}`;
   }
 
+  const includePathOptions =
+    includePaths === undefined ? '' : `--include-path ${includePaths.join(' --include-path ')}`;
+
   return {
     result: JSON.parse(
-      execSync(`${solcCommand} --standard-json ${allowedPaths}`, {
-        input: JSON.stringify(input),
-        maxBuffer: MAX_BUFFER_SIZE,
-        stdio: ['pipe', 'pipe', 'ignore'],
-      }).toString(),
+      execSync(
+        `${solcCommand} --standard-json ${allowedPaths} ${includePathOptions} --base-path .`,
```

Comment:
We should also just add base path as an option. Actually can you do an overview of all the options solc provides and make sure all the useful one's are threaded through correctly?